.github/mergify.yml

```diff
@@ -0,0 +1,80 @@
+---
+defaults:
+  actions:
+    queue:
+      name: default
+      method: rebase
+      update_method: rebase
+
+
+# each test should be listed separately, do not use regular expressions:
+# https://docs.mergify.io/conditions.html#validating-all-status-check
+# TODO: Use mergify's recently added 'shared configuration support'
+# to dedup some of the check-x=y repetition in the future.
+queue_rules:
+  - name: default
+    conditions:
+      - check-success=check
+        - check-success=build
+      - check-success=podmanbuild
+      - check-success=dockerbuild
+      - check-success=test
+      - check-success=test-kubernetes
+      -check-success=centos-ci/sink-clustered/mini-k8s-1.25
+
+        - check-success=dpulls
```

Comment:
List of checks are not aligned correctly and space after initial hyphen(`-`) is still not present. See `-check-success=centos-ci/sink-clustered/mini-k8s-1.25`

Reply:
fixed, thanks!